Jint/Runtime/Interop/DelegateWrapper.cs

```diff
@@ -37,6 +38,12 @@ public DelegateWrapper(Engine engine, Delegate d)
         public override JsValue Call(JsValue thisObject, JsValue[] jsArguments)
         {
             var parameterInfos = _d.Method.GetParameters();
+
+            if (parameterInfos.Length > 0 && parameterInfos[0].ParameterType.FullName == "System.Runtime.CompilerServices.Closure")
```

Comment:
Is typeof possible here?

Reply:
Have your tried wrapping the code with
```
#if NETFRAMEWORK
... the code ...
#endif
```

Reply:
I had only done it on the addition before - I just applied it to both and now it works, but it is basically ignoring the test case, which is probably not desirable. Does it get tested with both flags?

Reply:
Could you please add this line to appveyor.yml which is in the root folder:

```
- dotnet test .\Jint.Tests\Jint.Tests.csproj -c Release -f net452
```

That should include the test in CI builds.

Reply:
Done 👍